src/main/java/org/apache/cassandra/sidecar/cluster/CassandraAdapterDelegate.java

```diff
@@ -413,24 +413,28 @@ public TableOperations tableOperations()
     @Override
     public void onAdd(Host host)
     {
+        LOGGER.info("Host added. host={}", host);
```

Comment:
are these needed at info level? maybe debug?

Reply:
I think it is useful to log the lifecycle events at info. They should now appear too often. 
They are not related to the patch. I am open to remove them from the patch. Otherwise, I will keep them as is.

Reply:
My concern is from something I've seen in the past which is cql connection flapping. That's when log will become excessive . For that reason I'd rather have this at debug level and only enable it when needed